README.md

```diff
@@ -24,12 +24,12 @@ dex consists of multiple components:
 	- provides an API for bootstrapping the system.
 - **dexctl** is a CLI tool used to manage a dex deployment
 	- configure identity provider connectors
-	- administer OIDC client identities
+	- administer OIDC client identities.
```

Comment:
Indeed, not a complete sentence, but I went for consistency with the elements above. Thanks!